domain/src/main/java/edu/nextstep/camp/calculator/domain/HistoryItem.kt

```diff
@@ -0,0 +1,7 @@
+package edu.nextstep.camp.calculator.domain
+
+internal data class HistoryItem(val formula: Expression, val result: Expression) {
+    override fun toString(): String {
+        return "${formula}$NEW_LINE$EQUAL$SPACE${result}"
```

Comment:
```suggestion
        val historyFormat = "$formula\n= $result"
        return historyFormat
```
앗 제가 의도한 것은 이 것이 아니였는데, 제가 `상수`라는 키워드를 드려서 혼란을 드린 것 같네요 😭 
하드 코딩한 값을 변수로 정의하는 것을 의도했어요 🙏

Reply:
> 앗 제가 의도한 것은 이 것이 아니였는데, 제가 `상수`라는 키워드를 드려서 혼란을 드린 것 같네요 😭 하드 코딩한 값을 변수로 정의하는 것을 의도했어요 🙏

아~ 그랬군요. 
상수로 만들면서 전부 이렇게 치환하는게 과연 가독성이 좋아지는 방향인가? 라는 질문과 함께 많은 고민을 했었는데..ㅋㅋㅋ

괜찮습니다.^^ 아래 질문에 대한 답변은 항상 궁금했었거든요.
정말 도움되는 답변입니다.